actions/pull.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  init: () => 'NOT_PULLED',
+  startPulling: (store, _, actions) => {
+    fetch('/puller/status')
+      .then(resp => resp.json())
+      .then(resp => actions.setData(resp));
+    return 'PULLING'
+  },
+  setData: () => 'PULLED'
+}
```

Comment:
this is a dedicated action reducer which is only responsible for whether we need to fetch or not. calling `setData` sets the status to `'PULLED'`